pkg/openthread/contrib/platform_functions_wrapper.c

```diff
@@ -19,6 +19,7 @@
 
 #include <stdint.h>
 #include <stdio.h>
+#include <string.h>
```

Comment:
This was broken too?

Reply:
Well the header seem to have been included by `net/gnrc/netdev.h` (or one of the headers it included). When I switched that header out for `net/netdev.h`, I got lots of undefined functions for `memcpy`, `strcmp`, etc in this file ;-).

Reply:
I suspected something like that.